invenio_notifications/errors.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2023 Graz University of Technology.
+#
+# Invenio-Notifications is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Errors used in notification system."""
+
+from flask_babelex import gettext as _
+
+
+class NotificationError(Exception):
+    """General notification."""
+
+
+class NotificationBackendNotFoundError(NotificationError):
+    """The provided backend is not configured."""
+
+    def __init__(self, backend_id):
+        """Constructor.
+
+        :param backend_id: The id of the backend.
+        """
+        super().__init__(
+            _("Notification backend `{}` is not registered.".format(backend_id))
```

Comment:
I do not recall 100% the implication of formatting inside the extraction, but didnt we needed to format _after_ marking the string?

```suggestion
            _("Notification backend `{}` is not registered.").format(backend_id)
```

If yes, then it applies in more places.

Reply:
Thanks, will update